react/README.md

```diff
@@ -2,6 +2,9 @@
 
 *A mostly reasonable approach to React and JSX*
 
+This style guide is based on the standards that are currently prevalent in JavaScript or proposals in stage 3 or greater.
```

Comment:
Let's qualify this by saying it's case-by-case; async/await is still prohibited by this guide which is stage 4, as are class instance fields which are stage 3.

Reply:
Fair point. I've went ahead and improved the general idea of this comment by amending the last commit. Let me know how it looks like now.